WebRoot/css/zm.css

```diff
@@ -4021,23 +4021,69 @@ HTML>BODY .appt-selected .appt_allday_body
 }
 
 /* ZmChooseFolderDialog and ZmPickTagDialog */
+
+#ChooseFolderDialog {
```

Comment:
is #ChooseFolderDialog and .ChooserDialog pointing to same element?
if yes then can we use .ChooserDialog only?